lib/graph/make_graph.js

```diff
@@ -34,16 +34,41 @@ module.exports = function(config, options){
 		localSteal.System.config(options.localStealConfig);
 	}
 
+	// extend the loader with methods to add configuration required
+	// by the slim loader on runtime
+	(function() {
+		var slimLoaderConfig = {};
+
+		Object.defineProperties(slimLoaderConfig, {
+			extensions: {
+				value: [],
+				enumerable: true
+			},
+			toMap: {
+				value: [],
+				enumerable: true
+			},
+			identifiersToResolve: {
+				value: [],
+				enumerable: true
+			}
+		});
+
+		localSteal.System.normalizeMap = {};
+		localSteal.System.slimConfig = slimLoaderConfig;
```

Comment:
I first thought about hiding these being some getter/setters, but I was not sure about the API and ended up just using POJOs. Thoughts?